connector-webhook/src/main/java/com/redhat/cloud/notifications/connector/webhook/CloudEventHistoryBuilder.java

```diff
@@ -6,28 +6,32 @@
 import org.apache.camel.Exchange;
 import org.apache.camel.Message;
 
-import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.DISABLE_ENDPOINT_CLIENT_ERRORS;
-import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.HTTP_STATUS_CODE;
-import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.INCREMENT_ENDPOINT_SERVER_ERRORS;
+import static com.redhat.cloud.notifications.connector.http.ExchangeProperty.HTTP_CLIENT_ERROR;
+import static com.redhat.cloud.notifications.connector.http.ExchangeProperty.HTTP_SERVER_ERROR;
+import static com.redhat.cloud.notifications.connector.http.ExchangeProperty.HTTP_STATUS_CODE;
 
 @ApplicationScoped
 public class CloudEventHistoryBuilder extends OutgoingCloudEventBuilder {
 
+    public static final String STATUS_CODE = "HttpStatusCode";
+    public static final String DISABLE_ENDPOINT_CLIENT_ERRORS = "disableEndpointClientErrors";
+    public static final String INCREMENT_ENDPOINT_SERVER_ERRORS = "incrementEndpointServerErrors";
+
     @Override
     public void process(Exchange exchange) throws Exception {
         super.process(exchange);
-        boolean clientError = exchange.getProperty(DISABLE_ENDPOINT_CLIENT_ERRORS, false, boolean.class);
-        boolean serverError = exchange.getProperty(INCREMENT_ENDPOINT_SERVER_ERRORS, false, boolean.class);
+        boolean clientError = exchange.getProperty(HTTP_CLIENT_ERROR, false, boolean.class);
```

Comment:
Returning the HTTP error shouldn't be part of common-http ?

Reply:
As discussed in Meet, we'll take care of that in a subsequent PR.